lib/terraform/client.rb

```diff
@@ -50,8 +50,8 @@ def output(name:)
     end
 
     def version
-      execute command: factory.version_command do |value|
-        return ::Terraform::Version.create value: value
+      @version ||= execute command: factory.version_command do |value|
```

Comment:
An alternative solution, possibly more idiomatic to Test Kitchen, would be to attempt to fetch the version from the Test Kitchen state hash and use the client to retrieve it if it is not yet defined. This method would probably go to the `::Terraform::Configurable` dumping ground until it finds a better home. ^_^